docs/design/field-references.md

```diff
@@ -57,3 +57,65 @@ spec:
           name: sleeper3
         spec: "{{{sleeper2#spec}}}"
 ```
+
+## Referring to ServiceBinding outputs
+
+When Service Catalog processes a ServiceBinding, the output is placed in a Secret
```

Comment:
Could you put backticks around object kinds to make them look consistent with other files?